site/content/pages/organizing/01-Getting started/04-requirements.txt

```diff
@@ -0,0 +1,18 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Rules
+---
+
+Every devopsdays event is different, but there are a few rules to keep in mind if you'd like your event to be listed on devopsdays.org:
+
+- Inclusiveness and respect for differences are core devops values, and we invite you to help us make each devopsdays event a place that is welcoming and respectful to all participants. Your event will need to have a code of conduct. 
+- These are community events, so your event must accept registrations from the general public. Internal devops events focused on a specific organization are wonderful and we encourage them, but they won't be listed as "devopsdays" events on this site.
+- These events are not for individual or corporate profit. If you have money left over, you can use it for your next event, to help other devopsdays events, or for charity. The global core team does not have any ability to accept your money, but neither should it be making its way into anyone's pockets. :)
```

Comment:
> sharing the financial info back with the core team

Yeah, outside the scope here. Bring it up, by all means, if interested.

> I'd probably drop the smiley face here also, in case it is misuderstood

Done, and added a link to charity suggestions.